config.dev.toml

```diff
@@ -15,6 +15,11 @@ GoBinary = "go"
 # Env override: GO_ENV
 GoEnv = "development"
 
+# GoProxy specifies GOPROXY env for list/download command
+# which can be configured totally same with GOPROXY of Go Command
+# e.g. <proxy1>,<proxy2>,direct
```

Comment:
Forgot to mentionn: we should note that the comma-separated feature are only available in Go 1.13 and higher since the Dockerfile still uses Go 1.12

Reply:
fixed it.